homeassistant/components/smartthings/__init__.py

```diff
@@ -125,6 +125,13 @@
     return all(await asyncio.gather(*tasks))
 
 
+async def async_migrate_entry(hass, entry, version):
```

Comment:
I don't think that we need to pass in `version`, we know our own current version

Reply:
You’d hope.  :) I’ll remove.